src/uwtools/tests/config/formats/test_base.py

```diff
@@ -180,24 +180,6 @@ def test_parse_include(config):
     assert len(config["config"]) == 2
 
 
-@pytest.mark.parametrize("fmt1", [FORMAT.ini, FORMAT.nml, FORMAT.yaml])
-@pytest.mark.parametrize("fmt2", [FORMAT.ini, FORMAT.nml, FORMAT.yaml])
-def test_transform_config(fmt1, fmt2, tmp_path):
-    """
-    Test that transforms config objects to objects of other config subclasses.
```

Comment:
We don't want to do arbitrary transformations between formats anymore.

My UW-419 PR will add a unit test for YAML -> [INI, NML, SH] transformations, which we _do_ want to support.